docs/cli/getting-started.mdx

```diff
@@ -19,27 +19,15 @@ import TabItem from "@theme/TabItem";
 <TabItem value="macos" label="MacOS" default>
 
 ```shell
-curl -Lo botkube https://github.com/kubeshop/botkube/releases/download/v1.1.1/botkube_darwin_amd64
-```
-
-Copy to binary directory:
-
-```shell
-chmod +x botkube && mv botkube /usr/local/bin/botkube
+brew install kubeshop/botkube/botkube
 ```
 
 </TabItem>
 
 <TabItem value="linux" label="Linux">
 
 ```shell
-curl -Lo botkube https://github.com/kubeshop/botkube/releases/download/v1.1.1/botkube_linux_amd64
-```
-
-Copy to binary directory:
-
-```shell
-chmod +x botkube && mv botkube /usr/local/bin/botkube
+brew install kubeshop/botkube/botkube
```

Comment:
I suspect not many linux users use brew on linux, I'd put bin download here.

Reply:
fixed 👌